primitives/src/block.rs

```diff
@@ -207,6 +208,24 @@ impl Header {
     }
 }
 
+impl fmt::Display for Header {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            format_args!(
```

Comment:
Why? Unless I'm misremembering, `{}` inside `format_args` already removes formatting options.

Reply:
You are correct, fixed now.